client/app/page.tsx

```diff
@@ -1,20 +1,41 @@
 'use client'
-import { Button, Heading } from '@/_components/ui'
-import { useGetMeQuery } from '@/_hooks/api'
+
+import { Alert as NoHouseAlert, TodosList } from './(todos)/components'
+import { useGetTodosQuery } from './(todos)/hooks'
+import { Loading } from './_components/layout'
+import { Button, Heading } from './_components/ui'
 
 export default function Home() {
-  // This is the sample. Please change it to todos API query.
-  const { data } = useGetMeQuery()
+  const { data: todos, isError, isLoading } = useGetTodosQuery()
+
+  if (isLoading) {
+    return <Loading isCenter />
+  }
+
+  // TODO: Replace this with the error component
+  if (isError) {
+    return <p>Failed to load data.</p>
+  }
+
+  if (!todos) {
```

Comment:
Because there is no houseId, the `useGetTodoQuery` does not run, right? It means this would never show! You also need to wrap with div container!

Reply:
`useGetTodoQuery` doesn't run when there is no houseId, but it means todos become undefined, so in here I'm showing NoHouseAlert component when todos is undefined, which means the user is not in any house

I thought it's would be a good idea, but when the user has no house, they don't necessarily have to create a house and they can just wait for an invitation, so I think showing NoHouseAlert makes more sense.

Reply:
I deleted the Alert component!
https://github.com/HAK2024/fairy-share/pull/71/commits/38d451546e7f85f67065d9ffda6c7ee1c0ea268b